pages/about/contribute.md

```diff
@@ -9,10 +9,10 @@ Imagine a shared library of dynamic simulations – transparent, high-quality, a
 
 [Set up your Git environment and clone the project](#set-up-your-git-environment-and-clone-the-project)
```

Comment:
there is clone the project in the title but not in the description, we should add the git clone command of the repo

Reply:
Incorporated